db/init.sql

```diff
@@ -0,0 +1,41 @@
+ CREATE TABLE IF NOT EXISTS subscription(
+    subscription_id SERIAL PRIMARY KEY,
+    name  VARCHAR(100) NOT NULL,
+    duration INTERVAL NOT NULL,
+    cost INTEGER NOT NULL
+    ); 
+
+
+CREATE TABLE IF NOT EXISTS users(
+    user_id SERIAL PRIMARY KEY,
```

Comment:
keep it as just id, if the id is of the same table. Only when referring as foreign key, we need to name like this.